<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>741</width>
    <height>430</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item>
     <widget class="QWidget" name="widget" native="true">
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>100</height>
       </size>
      </property>
      <widget class="QPushButton" name="btnGo">
       <property name="geometry">
        <rect>
         <x>550</x>
         <y>70</y>
         <width>87</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string>Go</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="textDir">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>20</y>
         <width>131</width>
         <height>20</height>
        </rect>
       </property>
      </widget>
      <widget class="QToolButton" name="btnDir">
       <property name="geometry">
        <rect>
         <x>150</x>
         <y>20</y>
         <width>28</width>
         <height>22</height>
        </rect>
       </property>
       <property name="text">
        <string>...</string>
       </property>
      </widget>
      <widget class="QComboBox" name="textExt">
       <property name="geometry">
        <rect>
         <x>200</x>
         <y>20</y>
         <width>91</width>
         <height>22</height>
        </rect>
       </property>
       <property name="editable">
        <bool>true</bool>
       </property>
      </widget>
      <widget class="QComboBox" name="textSearch">
       <property name="geometry">
        <rect>
         <x>310</x>
         <y>20</y>
         <width>291</width>
         <height>22</height>
        </rect>
       </property>
       <property name="editable">
        <bool>true</bool>
       </property>
      </widget>
      <widget class="QCheckBox" name="checkRecurse">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>50</y>
         <width>79</width>
         <height>18</height>
        </rect>
       </property>
       <property name="text">
        <string>Recursive</string>
       </property>
      </widget>
      <widget class="QCheckBox" name="checkMatchCase">
       <property name="geometry">
        <rect>
         <x>200</x>
         <y>50</y>
         <width>101</width>
         <height>18</height>
        </rect>
       </property>
       <property name="text">
        <string>Match Case</string>
       </property>
      </widget>
      <widget class="QCheckBox" name="checkWholeWords">
       <property name="geometry">
        <rect>
         <x>310</x>
         <y>50</y>
         <width>161</width>
         <height>18</height>
        </rect>
       </property>
       <property name="text">
        <string>Exact phrase</string>
       </property>
      </widget>
      <widget class="QPushButton" name="btnAbout">
       <property name="geometry">
        <rect>
         <x>640</x>
         <y>70</y>
         <width>87</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string>About</string>
       </property>
      </widget>
      <widget class="QLabel" name="label">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>4</y>
         <width>59</width>
         <height>14</height>
        </rect>
       </property>
       <property name="text">
        <string>Directory</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_2">
       <property name="geometry">
        <rect>
         <x>200</x>
         <y>4</y>
         <width>101</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>File Extensions</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_3">
       <property name="geometry">
        <rect>
         <x>310</x>
         <y>4</y>
         <width>91</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Search Text</string>
       </property>
      </widget>
      <widget class="QPushButton" name="btnSave">
       <property name="geometry">
        <rect>
         <x>460</x>
         <y>70</y>
         <width>87</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string>Save</string>
       </property>
      </widget>
     </widget>
    </item>
    <item>
     <widget class="QTableWidget" name="listResults">
      <property name="editTriggers">
       <set>QAbstractItemView::NoEditTriggers</set>
      </property>
      <property name="selectionMode">
       <enum>QAbstractItemView::SingleSelection</enum>
      </property>
      <property name="selectionBehavior">
       <enum>QAbstractItemView::SelectRows</enum>
      </property>
      <attribute name="horizontalHeaderVisible">
       <bool>false</bool>
      </attribute>
      <attribute name="verticalHeaderVisible">
       <bool>false</bool>
      </attribute>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
